entities/Base.java

```diff
@@ -15,6 +15,10 @@ public abstract class Base {
     private Long createdBy;
     private Long updatedBy;
     private Status status;
+    private String firstName;
+    private String lastName;
+
+
```

Comment:
I don't think all entities will have a `firstName` and a `lastName`.

What do youi think about adding a new PR in order to make something like this possible:
```
puiblic class Chef extends User/Person {
.
.
.
}

public class User/Person {
firstName;
lastName;
nationalId;
age;
.
.
.
}
```

Looks like Chef and Customers are/have same properties as User/Person